templates/modern/src/nav.ts

```diff
@@ -131,6 +130,9 @@ function findActiveItem(items: (NavItem | NavItemContainer)[]): NavItem {
   let activeItem: NavItem
   let maxPrefix = 0
   for (const item of items.map(i => 'items' in i ? i.items : i).flat()) {
+    if (item.href.hostname !== window.location.hostname || item.href.port !== window.location.port) {
```

Comment:
consider move `isExternal` to `helper.ts` and reuse the function here?

Reply:
Will do